io/io/src/TFile.cxx

```diff
@@ -4030,6 +4036,18 @@ TFile *TFile::Open(const char *url, Option_t *options, const char *ftitle,
       return f;
    }
 
+#ifdef R__UNIX
+   { // If the file has an extended attribute with the XRootD URL, try it first
+      ssize_t len = getxattr(url, "eos.url.xroot", nullptr, 0);
```

Comment:
should probably add a comment here mentioning that `getxattr` only works with plain paths and if `url` starts with `file://` (or any protocol for what that matters, if I understand correctly) `getxattr` will fail and we'll skip the rest of the logic.

Reply:
Yes, I guess the comment I added is not enough. I will update it with your suggestion.